openapi.yaml

```diff
@@ -25,254 +37,677 @@ security:
+          type: string
+          description: "ID of the dough to use"
+        - in: path
+          name: pizzaID
+          required: true
+          type: string
+          description: "ID of a pizza to update"
+      responses:
+        200:
+          description: "Returns true is the resource was updated"
+          schema:
+            type: boolean
+            example: true
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
+  /doughs:
+    get:
+      description: "Get all dough recipes"
+      tags:
+        - dough
+      parameters:
+        - in: query
+          name: name
+          required: false
+          type: string
+          description: Search doughs by name
+        - in: query
+          name: gramsFlour
+          required: false
+          type: integer
+          description: Search doughs by grams flour used
+        - in: query
+          name: gramsWater
+          required: false
+          type: integer
+          description: Search doughs by grams water used
+        - in: query
+          name: gramsSugar
+          required: false
+          description: Search doughs by grams sugar used
+          type: integer
+        - in: query
+          name: gramsOliveOil
+          required: false
+          description: Search doughs by grams olive oil used
+          type: integer
+        - in: query
+          name: gramsYeast
+          required: false
+          type: integer
+          description: Search doughs by grams yeast used
+        - in: query
+          name: gramsSalt
+          required: false
+          type: integer
+          description: Search doughs by grams salt used
+        - in: query
+          name: bulkFermentTime
+          required: false
+          type: integer
+          description: Search doughs by bulk ferment time
+        - in: query
+          name: proofTime
+          required: false
+          type: integer
+          description: Search doughs by proof time
+        - in: query
+          name: waterTemp
+          required: false
+          type: integer
+          description: Search doughs by water temperature at mix time
+        - in: query
+          name: flourType
+          required: false
+          description: Search doughs by flour type
+          type: string
+      responses:
+        200:
+          description: "A list of dough recipes"
+          schema:
+            $ref: "#/definitions/DoughsResult"
+        500:
+          $ref: "#/responses/500"
+    post:
+      description: "Create a dough recipe"
+      tags:
+        - dough
+      parameters:
+        - in: body
+          name: attributes
+          description: attributes of the new dough recipe
+          schema:
+            $ref: "#/definitions/DoughAttributes"
+      responses:
+        200:
+          description: "The created dough recipe"
+          schema:
+            $ref: "#/definitions/DoughResult"
+        500:
```

Comment:
Besides `200` and `500`, can you think of any possible status codes?

Reply:
This would also make sense to have a `400` response